<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectLocksWidget</class>
 <widget class="QWidget" name="SelectLocksWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Locks</string>
  </property>
  <widget class="QLabel" name="lblSelectedLocks">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>11</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected Locks:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstSelectedLocks">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>41</y>
     <width>221</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenSelected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>351</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vloLockCabinet"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
